test/generators/typescript/renderers/EnumRenderer.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { TypeScriptGenerator } from '../../../../src/generators';
+import { EnumRenderer } from '../../../../src/generators/typescript/renderers/EnumRenderer';
+import { CommonInputModel, CommonModel } from '../../../../src/models';
+
+describe('TypeScriptRenderer', () => {
```

Comment:
Ups, forgot to change `TypeScriptRenderer` to `EnumRenderer` 😅

Reply:
Yep, I did have a thought about it 😬

Reply:
Haha, trust yourself when you notice something like that (regardless of who gives you the code) and raise the flag 👍

Reply:
Aye aye captain!

Reply:
@Ishan-Saini I dont see the change to the test name? DId I miss the commit or?

Reply:
Oops, my apologies. I pushed the changes.
It was a silly mistake.